modules/Bio/EnsEMBL/Variation/Pipeline/SpliceAI/SpliceAI_conf.pm

```diff
@@ -0,0 +1,195 @@
+        hive_force_init      => 1,
+        hive_use_param_stack => 1,
+
+        pipeline_name           => $self->o('pipeline_name'),
+        main_dir                => $self->o('main_dir'), # main_dir = '/hps/nobackup2/production/ensembl/dlemos/tmp_spliceai_pipeline'
+        tmp_split_vcf_dir       => $self->o('main_dir') . '/tmp_split_vcf', # contains vcf input splited without headers
+        tmp_split_vcf_input_dir => $self->o('main_dir') . '/tmp_split_vcf_input', # contains vcf input splited with headers, these are the files used to run SpliceAI
+        tmp_output_dir          => $self->o('main_dir') . '/tmp_output', # temporary output files, still splited
+        output_dir              => $self->o('main_dir') . '/output', # final output files already merged by chromosome
+        fasta_file              => $self->o('fasta_file'), # '/hps/nobackup2/production/ensembl/dlemos/files/Homo_sapiens.GRCh38.dna.toplevel.fa'
+        gene_annotation         => $self->o('gene_annotation'), # '/homes/dlemos/work/tools/SpliceAI_files_output/gene_annotation/ensembl_gene/grch38_MANE_8_7.txt'
+        step_size               => $self->o('step_size'), # number of variants used to split the main vcf files
+        check_transcripts       => $self->o('check_transcripts'), # checks which are the new MANE transcripts for the last months, runs SpliceAI only for these ones
+        registry                => $self->o('registry'), # database where new MANE transcripts are going to be checked
+        # transcripts_file        => $self->o('transcripts_file'), # Instead of checking the new transcripts from the database, it can run only for transcripts from this file
+        output_file_name        => 'spliceai_scores_',
+
+        pipeline_wide_analysis_capacity => 100,
+
+        pipeline_db => {
+            -host   => $self->o('hive_db_host'),
+            -port   => $self->o('hive_db_port'),
+            -user   => $self->o('hive_db_user'),
+            -pass   => $self->o('hive_db_password'),
+            -dbname => $ENV{'USER'} . '_hive_' . $self->o('pipeline_name'),
+            -driver => 'mysql',
+        },
+    };
+}
+
+sub resource_classes {
+    my ($self) = @_;
+    return {
+        %{$self->SUPER::resource_classes},
+        '4Gb_8c_job'  => {'LSF' => '-n 8 -q production-rh74 -R"select[mem>4000]  rusage[mem=4000]" -M4000' },
+        '4Gb_job'     => {'LSF' => '-q production-rh74 -R"select[mem>4000] rusage[mem=4000]" -M4000'},
+    };
+}
+
+sub pipeline_analyses {
+  my ($self) = @_;
+  my @analyses;
+  push @analyses, (
+      # pre run checks, directories exist etc
+
+      {   -logic_name => 'init_files',
+          -module     => 'Bio::EnsEMBL::Hive::RunnableDB::JobFactory',
+          -input_ids  => [{}],
+          -parameters => {
+            'input_directory' => $self->o('input_directory'),
+            'inputcmd'        => 'find #input_directory# -type f -name "all_snps_ensembl_38_*.vcf" -printf "%f\n"',
```

Comment:
This contains the assembly version 38. Can this not also be run for 37?

Reply:
It can run for both. I'll delete the assembly.